### PR TITLE
LPD-41141 Create Exception Mapper for Object Relationship Edge Exceptions

### PR DESCRIPTION
**Pull request by victorKammerer**
Jira ticket: [LPD-41141](https://liferay.atlassian.net/browse/LPD-41141)

[LPD-41141]: https://liferay.atlassian.net/browse/LPD-41141?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by victorKammerer** [comment]
ci:test:sf

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Hey @victorKammerer, we have to create the language keys and use them inside the exceptions as we discussed, also remember to check with the frontend guys if the messages in the backend are matching the expected messages in the mockups.

Thank you!

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java [hunk 1]
```diff
@@ -15,8 +15,10 @@ public class ObjectRelationshipEdgeException extends PortalException {
 	public ObjectRelationshipEdgeException() {
 	}
 
-	public ObjectRelationshipEdgeException(String msg) {
+	public ObjectRelationshipEdgeException(String msg, String messageKey) {
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java:18]: Hey @victorKammerer could you please rename `String msg` to `String message` to keep the pattern used in other exceptions like this [one](https://github.com/liferay/liferay-portal/blob/1cfd18f9cbc0e7be4f28ccd2da369ed6f0b1620d/modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectActionNameException.java#L76) for example.

Thank you!

**Review by MarinhoFeliphe — APPROVED** [review]
@victorKammerer just this rename. Good job! :tada: 

Thank you!

**Comment by victorKammerer** [comment]
ci:forward

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java (hunk 1 above)
Comment (victorKammerer) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java]: Done!

**Comment by victorKammerer** [comment]
ci:forward

**Comment by victorKammerer** [comment]
Relevant tests failures seems unrelated, could you please force forward? @MarinhoFeliphe

**Comment by victorKammerer** [comment]
ci:forward:force

**Pull request closed without merging**